spec/unit/kiwi_config_spec.rb

```diff
@@ -224,6 +231,21 @@
       expect(config.sh).not_to match(/systemctl \w+-runtime /)
     end
 
+    context "if an empty docker service scope is provided" do
+      it "does not raise" do
```

Comment:
Do you think it's necessary to have this test? It's implicitly tested in the `shows a warning` test anyway.